apps/frontend-v3/lib/vebal/useVebalUserData.ts

```diff
@@ -22,12 +27,24 @@ export function useVebalUserData() {
     balanceResponse.refetch()
   }
 
+  const snapshots = apiResponse.data?.veBalGetUser.lockSnapshots
+  const lastLockTimestamp =
+    snapshots && snapshots.length > 0
+      ? bn(calculateLastLock(snapshots).timestamp).times(1000).toNumber()
```

Comment:
you can do `fromUnixTime(calculateLastLock(snapshots).timestamp).getTime()` to remove the magic number

Reply:
Have ended up using `secondsToMilliseconds`.